packages/components/src/Drawer/DrawerSurface.tsx

```diff
@@ -43,7 +47,7 @@ export interface DrawerSurfaceProps {
    * the specified width or the viewport width.
    * @default 'medium'
    */
-  width?: ResponsiveValue<DialogSizeRamp | string>
+  width?: DrawerWidth
```

Comment:
This needs to support `string` still as folks may want to use "off standard" sizes:

```suggestion
  width?: DrawerWidth | string
```

Also removing `ResponsiveValue` will act as a breaking change for anybody using that so we need to put it back so as not to break folks who might be using responsive drawer sizes (not sure if anyone is but it's hard to know for sure since extension developers use stuff in new ways).